slice/IceGrid/Registry.ice

```diff
@@ -109,9 +109,11 @@ module IceGrid
         AdminSession* createAdminSessionFromSecureConnection()
             throws PermissionDeniedException;
 
-        /// TODO: update description or remove operation
+        /// Gets the session timeout. An Ice 3.7 or earlier client can use this value to determine how often it needs to
+        /// send heartbeats (using ACM) or call {@link Session#keepAlive} (resp. {@link AdminSession#keepAlive}) to keep
+        /// a session alive in the IceGrid registry.
```

Comment:
Why would a 3.7 want to call keepAlive? I assume is just to keep the connection alive as the impl does nothing.